### PR TITLE
Chore: escape quotes in release action

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves

The release action pastes the contents of the RC pull request into a bash variable using an echo command, but if the PR description contains a quote it breaks the bash syntax.

E.g.
```
Remove "What's new"
```

### CODE REVIEW

File: .github/workflows/tag-release.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Tag & GitHub release
 on:
   pull_request_target:
     branches:
-      - main
+      - dev
```
Comment (usame-algan) [inline .github/workflows/tag-release.yml:6]: Why the change from main to dev?

**Comment by katspaugh** [comment]
After trying a few different things, the best solution was to pass the body as a variable instead of a file. For that, I've replaced our release action which also fixed the obsolete set_output warnings.
